service/resource/configmap/resource.go

```diff
@@ -253,7 +253,7 @@ func (r *Resource) ProcessDeleteState(ctx context.Context, obj, deleteState inte
 
 		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "deleted the config maps in the Kubernetes API")
 	} else {
-		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "the config maps do not exist in the Kubernetes API")
+		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "the config maps do not need to be deleted from the Kubernetes API")
```

Comment:
The current logs look like this, which is pretty useless. 

```
{"caller":"github.com/giantswarm/kvm-operator/service/resource/namespace/resource.go:205","cluster":"ntt2j","debug":"the namespace does already exist in the Kubernetes API","resource":"namespace","time":"17-09-15 16:33:29.831"}
{"caller":"github.com/giantswarm/kvm-operator/service/resource/namespace/resource.go:233","cluster":"ntt2j","debug":"the namespace does not exist in the Kubernetes API","resource":"namespace","time":"17-09-15 16:33:29.831"}
```

The messages are now improved.